### DISCUSSION TITLE
put more than one picture on page!

### DISCUSSION THREAD

**Discussion by asdf10203040** [jspsych/jsPsych#1295] [Q&A]
is there ant way that we can put more than one picture on the html page, such as "html-keyboard-response" or "image-keyboard-response"? or other ?

**Comment by kschuler** [comment]
`<img src='url/to/img1'></img><img src='url/to/img2'></img>`

Yes, any HTML can go in to most prompt or html stim parameters. If you want to lay them out in a certain way, you can use divs. jsPsych uses the bootstrap css library, so this might help: https://getbootstrap.com/docs/4.1/layout/grid/

**Reply by kschuler** [reply]
Actually, I'm not sure if jsPsych natively has boostrap! You can add it like this

```
    <!-- stylesheets for making everything pretty -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">

    <link href="../assets/js/jspsych-6.1.0/css/jspsych.css" rel="stylesheet" type="text/css"></link>
```

You can add it above jsPsych so any settings jsPsych has that are special will be preferred.
